components/org.wso2.carbon.identity.oauth.rar/src/main/java/org/wso2/carbon/identity/oauth2/rar/AuthorizationDetailsSchemaValidator.java

```diff
@@ -0,0 +1,226 @@
+/*
+ * Copyright (c) 2024, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
```suggestion
 * Copyright (c) 2025, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Let's change in other places too